### PR TITLE
LPS-203459 As a user, I would like to a more specific error message instead of internal server error when endpoint and schema are not in same scope

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPS-203459

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:relevant

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java [hunk 1]
```diff
@@ -608,7 +608,7 @@ private List<ServiceRegistration<?>> _registerExceptionMappers(
 						"osgi.jaxrs.application.select",
 						StringBundler.concat(
 							"(|(liferay.objects.exception.mapper=true)",
-							"(osgi.jaxrs.name= ", jaxRsApplicationName, "))")
+							"(osgi.jaxrs.name=", jaxRsApplicationName, "))")
```
Comment (ccorreagg) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java:611]: :open_mouth:

Comment (magjed4289) [reply modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java:611]: yes, in the end it was a space matter, @4lejandrito has discovered it. Nonetheless, I was able to learn about different ExceptionMappers along the way, so I don't consider it a waste of time :D

**Comment by magjed4289** [comment]
ci:test:relevant

**Comment by magjed4289** [comment]
ci:test:sf

**Review by 4lejandrito — APPROVED** [review]
See my changes!

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java (hunk 1 above)
Comment (4lejandrito) [reply modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java:611]: :woman_astronaut:

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
@magjed4289 the SF failure is real.

**Comment by magjed4289** [comment]
@4lejandrito SF done! Will be forwarding, let's see if test pass

**Comment by magjed4289** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
ci:forward:force

**Pull request closed without merging**